ibm/provider/provider.go

```diff
@@ -179,6 +179,12 @@ func Provider() *schema.Provider {
 				Description: "The retry count to set for API calls.",
 				DefaultFunc: schema.EnvDefaultFunc("MAX_RETRIES", 10),
 			},
+			"code_engine_version": {
```

Comment:
In provider schema we will have only those attributes which is common for all services under IBM Cloud.
We can't have service specific config params at provider level

Reply:
Additional information on what we are attempting to do can be found here: https://ibm-cloudplatform.slack.com/archives/C4XC6SWKG/p1736451691992129

Please let us know once you have an idea on how to best handle this.